app/main/middlewares/log_page_view.py

```diff
@@ -0,0 +1,20 @@
+import json
+from functools import wraps
+
+from flask import current_app, request, session
+
+
+def log_page_view(route_function):
+    @wraps(route_function)
+    def wrapper(*args, **kwargs):
+        user_id = session.get("user_id", "anonymous")
+        log_data = {
+            "event": "page_view",
```

Comment:
some thoughs:

1. every authenticated route has this decorator and I think that makes sense but the download route is not a "page". 
2. the download route already has the audit log about the user downloading a file in a different structure
3. there is also route and method in the log data here. Does the event field add anything?

based on this I'm thinking we could just remove the event field and the existing audit log call for the user as long as the route includes the detail id for detail routes e.g. /download/id.

Worth me you and @colinbowen  having a chat about this